components/tabs/tab.js

```diff
@@ -32,21 +32,30 @@ class Tab extends TabMixin(LitElement) {
 				box-shadow: 0 0 0 2px var(--d2l-color-celestine);
 				color: var(--d2l-color-celestine);
 			}
+			.d2l-tab-text-skeletize-override {
+				min-width: 50px;
+			}
+			:host([skeleton]) .d2l-tab-text.d2l-skeletize::before {
+				bottom: 0.15rem;
+				top: 0.15rem;
+			}
+
 		`];
 
 		super.styles && styles.unshift(super.styles);
 		return styles;
 	}
 
 	renderContent() {
+		const overrideSkeletonText = this.skeleton && (!this.text || this.text.length === 0);
 		const contentClasses = {
 			'd2l-tab-text': true,
+			'd2l-skeletize': true,
+			'd2l-tab-text-skeletize-override': overrideSkeletonText
 		};
 
 		return html`
-			<div class="${classMap(contentClasses)}">
-				${this.text}
-			</div>
+			<div class="${classMap(contentClasses)}">${overrideSkeletonText ? html`&nbsp;` : this.text}</div>
```

Comment:
Curious thoughts on this - this leaves it to consumers of `TabMixin` to handle how they want to skeletizing. Could also just put it in the mixin and have one tab blob.

Reply:
I think this makes sense but could just rely on being falsy, and drop the variable
```suggestion
			<div class="${classMap(contentClasses)}">${this.text || html`&nbsp;`}</div>
```

Reply:
Checked with Jeff and he thinks it would be best if we handle the skeletization. I'll move this into `TabMixin`